arionBackend/tests/api/test_hierarchy.py

```diff
@@ -66,7 +66,7 @@ def test_create_hierarchy_success(self):
 		request = self.client.post(
 			'/api/hierarchy/esper/create',
 			follow=True,
-			data={"name": "Test", "queries": ["S"]},
+			data={"name": "Test", "queries": ["INSERT INTO asd SELECT asd FROM asd", "INSERT INTO asd SELECT asd FROM asd WHERE 1=1"]},
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') The line contains 126 characters which is greater than 120 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.tommartensen.de/coding_rules#rule_key=python%3ALineLength)